### PR TITLE
Updating exit code enums

### PR DESCRIPTION
**Pull request by stephanwampouille**
This PR aligns all exit codes with the new enums set in: https://docs.google.com/spreadsheets/d/1Feq6KEnXi-u14c0V-rVw0b4XzTFRbwPRFcMA7y3Td5M/edit#gid=1576963612

### CODE REVIEW

**Comment by stephanwampouille** [comment]
@privateducky @makk94 this is ready for review

**Comment by privateducky** [comment]
@makk94 letting you do a first pass.

File: python/sdk/prelude_sdk/models/codes.py [hunk 1]
```diff
@@ -45,26 +45,24 @@ def _missing_(cls, value):
...
+    BEHAVIOR_PREVENTED_3 = 107
+    TEST_UNAVAILABLE = 108
+    INCORRECTLY_BLOCKED = 110
+    BEHAVIOR_PREVENTED_4 = 126
+    SIGNATURE_PREVENTED_2 = 127
+    OUT_OF_MEMORY = 137
+    UNEXPECTED_ERROR = 256
 
     @classmethod
     def _missing_(cls, value):
```
Comment (makk94) [inline python/sdk/prelude_sdk/models/codes.py:68]: this method is now:
```

    @classmethod
    def _missing_(cls, value):
        if value and not isinstance(value, int):
            return cls(int(value))
        return ExitCode.MISSING
```

File: python/sdk/prelude_sdk/models/codes.py [hunk 2]
```diff
@@ -45,26 +45,24 @@ def _missing_(cls, value):
...
-    UNEXPECTED = 256
+    SIGNATURE_PREVENTED_1 = 105
+    BLOCKED_AT_PERIMETER = 106
+    BEHAVIOR_PREVENTED_3 = 107
+    TEST_UNAVAILABLE = 108
+    INCORRECTLY_BLOCKED = 110
+    BEHAVIOR_PREVENTED_4 = 126
+    SIGNATURE_PREVENTED_2 = 127
+    OUT_OF_MEMORY = 137
+    UNEXPECTED_ERROR = 256
```
Comment (makk94) [inline python/sdk/prelude_sdk/models/codes.py:66]: we also have a transform() method  that BE uses:
```

    def transform(self, test):
        if test.unit == 'health' and self != ExitCode.PROTECTED:
            return ExitCode.FALSE_POSITIVE
        return self
```

Comment (stephanwampouille) [reply python/sdk/prelude_sdk/models/codes.py:66]: I added it with the updated values

File: python/sdk/prelude_sdk/models/codes.py (hunk 1 above)
Comment (stephanwampouille) [reply python/sdk/prelude_sdk/models/codes.py:68]: adjusted

**Review by privateducky — APPROVED** [review]

**Pull request merged**